ci/jenkins/jobs/projects.yaml

```diff
@@ -811,6 +811,762 @@
           started_status: null
           wrappers: []
           publishers: []
+      - '{name}-{test_name}-no-scm':
```

Comment:
I think maybe we can have two projects.yaml files.

projects-cloud.yaml
```
- project:
    # ghpr_auth, antrea_admin_list, antrea_org_list and antrea_white_list
    # should be defined as a global variable somewhere else
    name: antrea
    git_credentials_id: ANTREA_GIT_CREDENTIAL
    org_repo: antrea-io/antrea
    jobs:
      # jobs which can be run on VMC
```

projects-lab.yaml
```
- project:
    name: antrea-lab  # use a different name here other than antrea
    ...
    jobs:
      # jobs which can be run on lab
```

When we update jobs on lab jenkins, we can rm projects-cloud.yaml. When we update jobs on public jenkins, we can rm projects-lab.yaml.

We can migrate jobs between cloud and lab by moving job between the two files.

Reply:
Thought about this twice. I think the two files can use the same project name. Firstly I thought jenkins-job-builder may not support two projects with same name. Now I think we will never jenkins-job update the two files at the same time. We always delete one of them, so it should be fine to use the same project name.

Reply:
Updated. Thanks for the suggestion!